### PR TITLE
Read term dates from manual source in ep-data repo

### PR DESCRIPTION
**Pull request by ondenman**
The end date for term 44 was incorrect, causing problems with the data.

### CODE REVIEW

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
You really need to include a source for this information in the commit message.

However, maintaining this list in multiple places is also providing problematic, so I'd say this should be entirely replaced with looking it up using everypolitician-ruby:

`EveryPolitician::Index.new.country('Australia').legislature('Representatives').legislative_periods`

**Comment by ondenman** [comment]
Term dates now read from manual file in ep-data repo.

File: Gemfile [hunk 1]
```diff
@@ -14,3 +14,4 @@ gem "nokogiri"
 gem "open-uri-cached"
 gem "fuzzy_match"
 gem 'wikidata-client', '~> 0.0.7', require: 'wikidata'
+gem 'everypolitician'
```
Comment (tmtmtmtm) [inline Gemfile]: I don't think you need to include this any more, do you?

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
You've removed the Gemfile, but not the Gemfile.lock

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
You still have what appear to be needless changes to `Gemfile.lock`

Rather than adding https://github.com/everypolitician-scrapers/australia-openaustralia/pull/4/commits/aafb75a43bdf96a4ba1e4e61be8f06550b064631 to revert https://github.com/everypolitician-scrapers/australia-openaustralia/pull/4/commits/c722a04d9cd683f11679d304f2954c128cf86c7d, I'd suggest you simply drop both commits.

**Comment by ondenman** [comment]
@tmtmtmtm I've rebased and removed both of the gemfile commits.

**Review by tmtmtmtm — APPROVED** [review]

**Pull request merged**